docs/api/structures/web-request-filter.md

```diff
@@ -1,4 +1,5 @@
 # WebRequestFilter Object
 
-* `urls` string[] - Array of [URL patterns](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Match_patterns) that will be used to filter out the requests that do not match the URL patterns.
-* `types` String[] (optional) - Array of types that will be used to filter out the requests that do not match the types. When not specified, all types will be matched. Can be `mainFrame`, `subFrame`, `stylesheet`, `script`, `image`, `font`, `object`, `xhr`, `ping`, `cspReport`, `media` or `webSocket`.
+* `urls` string[] - Array of [URL patterns](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Match_patterns) used to include requests that match these patterns. An empty array matches no URLs, while using `<all-urls>` matches all URLs.
```

Comment:
```suggestion
* `urls` string[] - Array of [URL patterns](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Match_patterns) used to include requests that match these patterns. An empty array matches no URLs, while using `<all_urls>` matches all URLs.
```

Reply:
"empty array matches no URLs" should be removed until we can fully deprecate that behavior.

Reply:
Oh yeah good call out! Will update!